### PR TITLE
More style and formatting change - in bitmessageqt

### PR DESCRIPTION
**Pull request by g1itch**
These two commits are originally from `ui-refactoring`, but there essentially style and formatting changes.

### CODE REVIEW

File: src/bitmessageqt/__init__.py [hunk 1]
```diff
@@ -1890,11 +1902,13 @@ def addRow (address, label, type):
...
-        for address in sorted(oldRows, key = lambda x: oldRows[x][2], reverse = True):
-            if address in newRows:
-                completerList.append(unicode(newRows[address][0], encoding="UTF-8") + " <" + address + ">")
-                newRows.pop(address)
-            else:
+        for address in sorted(
+            oldRows, key=lambda x: oldRows[x][2], reverse=True
+        ):
+            try:
+                completerList.append(
```
Comment (PeterSurda) [inline src/bitmessageqt/__init__.py]: Have you tested it with UTF-8 characters (e.g. cyrillic will probably be the easiest for you to test)?

File: src/bitmessageqt/__init__.py [hunk 2]
```diff
@@ -2279,26 +2296,43 @@ def rerenderComboBoxSendFromBroadcast(self):
...
-            elif not helper_search.check_match(toAddress, fromAddress, subject, message, self.getCurrentSearchOption(tab), self.getCurrentSearchLine(tab)):
+            elif not helper_search.check_match(
+                toAddress, fromAddress, subject, message,
+                self.getCurrentSearchOption(tab),
+                self.getCurrentSearchLine(tab)
+            ):
                 continue
-            
-            self.addMessageListItemSent(sent, toAddress, fromAddress, subject, "msgqueued", ackdata, time.time())
-            self.getAccountTextedit(acct).setPlainText(unicode(message, 'utf-8', 'replace'))
```
Comment (PeterSurda) [inline src/bitmessageqt/__init__.py]: Have you tested this with UTF-8?

File: src/bitmessageqt/__init__.py [hunk 3]
```diff
@@ -2279,26 +2296,43 @@ def rerenderComboBoxSendFromBroadcast(self):
...
+
+            self.addMessageListItemSent(
+                sent, toAddress, fromAddress, subject, "msgqueued",
+                ackdata, time.time())
+            self.getAccountTextedit(acct).setPlainText(message)
```
Comment (PeterSurda) [inline src/bitmessageqt/__init__.py]: Have you tested this with UTF-8?

File: src/bitmessageqt/__init__.py [hunk 4]
```diff
@@ -3656,33 +3706,29 @@ def getAccountTextedit(self, account):
...
-        ]
+        )
         if currentIndex >= 0 and currentIndex < len(messagelistList):
             if retObj:
                 return messagelistList[currentIndex]
             else:
-                return messagelistList[currentIndex].text().toUtf8().data()
-        else:
-            return None
+                return messagelistList[currentIndex].text().toUtf8()
```
Comment (PeterSurda) [inline src/bitmessageqt/__init__.py]: Have you tested this?

File: src/bitmessageqt/__init__.py (hunk 1 above)
Comment (g1itch) [reply src/bitmessageqt/__init__.py]: retested just now

File: src/bitmessageqt/__init__.py (hunk 4 above)
Comment (g1itch) [reply src/bitmessageqt/__init__.py]: oops! https://github.com/g1itch/PyBitmessage/commit/1eb3eb0ed5ab4f867abbad813fd9d6d70720d35a

**Comment by g1itch** [comment]
I've decided to remove the controversial commit. It is probably not suitable for v0.6 because it's originally from qt5-wip branch.

**Pull request merged**